<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>voix's team</author>
 <class>ITClient</class>
 <widget class="QWidget" name="ITClient">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>443</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>765</width>
    <height>443</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>765</width>
    <height>443</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>IT Client</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(43, 43, 43)</string>
  </property>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>17</y>
     <width>162</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white</string>
   </property>
   <property name="text">
    <string>IT Client</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Home_Button">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>16</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Light</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;background-color: transparent</string>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Version_button">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>17</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Light</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;background-color: transparent</string>
   </property>
   <property name="text">
    <string>Versions</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Profile">
   <property name="geometry">
    <rect>
     <x>618</x>
     <y>10</y>
     <width>135</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(130, 130, 130), stop:1 rgb(171, 171, 171));color: rgb(255, 255, 255);border-top-left-radius : 10px;border-top-right-radius : 10px;border-bottom-left-radius:10px;border-bottom-right-radius : 10px;

QPushButton {
    background-color: rgb(0, 98, 0)
}</string>
   </property>
   <property name="text">
    <string>CLICK TO LOGIN</string>
   </property>
  </widget>
  <widget class="QFrame" name="Versions">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>48</y>
     <width>828</width>
     <height>393</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(43, 43, 43)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <widget class="QPushButton" name="v189">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>50</y>
      <width>191</width>
      <height>37</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(125, 125, 125);color: rgb(255, 255, 255);border-top-left-radius : 10px;border-top-right-radius : 10px;border-bottom-left-radius:10px;border-bottom-right-radius : 10px;

QPushButton {
    background-color: rgb(0, 98, 0)
}</string>
    </property>
    <property name="text">
     <string>1.8.9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="v1122">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>7</y>
      <width>191</width>
      <height>37</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(125, 125, 125);color: rgb(255, 255, 255);border-top-left-radius : 10px;border-top-right-radius : 10px;border-bottom-left-radius:10px;border-bottom-right-radius : 10px;

QPushButton {
    background-color: rgb(0, 98, 0)
}</string>
    </property>
    <property name="text">
     <string>1.12.2</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="Home">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>48</y>
     <width>828</width>
     <height>399</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="start_game">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>72</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(36, 157, 0);color: rgb(255, 255, 255);border-top-left-radius : 10px;border-top-right-radius : 10px;border-bottom-left-radius:10px;border-bottom-right-radius : 10px;

QPushButton {
    background-color: rgb(0, 98, 0)
}</string>
    </property>
    <property name="text">
     <string>Launch Game</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>141</x>
     <y>36</y>
     <width>34</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>36</y>
     <width>34</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Home_Button</sender>
   <signal>clicked()</signal>
   <receiver>Versions</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Home_Button</sender>
   <signal>clicked()</signal>
   <receiver>Home</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Version_button</sender>
   <signal>clicked()</signal>
   <receiver>Versions</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Version_button</sender>
   <signal>clicked()</signal>
   <receiver>Home</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Home_Button</sender>
   <signal>clicked()</signal>
   <receiver>frame</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Version_button</sender>
   <signal>clicked()</signal>
   <receiver>frame</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Version_button</sender>
   <signal>clicked()</signal>
   <receiver>frame_2</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Home_Button</sender>
   <signal>clicked()</signal>
   <receiver>frame_2</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
